patches/content-browser-webauth-webauth_request_security_checker.cc.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/content/browser/webauth/webauth_request_security_checker.cc b/content/browser/webauth/webauth_request_security_checker.cc
+index 6922a3df8de2f76a03616450e9ad0c8205bf228a..47e37b709033b35a95cbcbb597a4bd07b27c1d52 100644
+--- a/content/browser/webauth/webauth_request_security_checker.cc
++++ b/content/browser/webauth/webauth_request_security_checker.cc
+@@ -81,6 +81,7 @@ WebAuthRequestSecurityChecker::RemoteValidation::~RemoteValidation() = default;
+ // static
+ std::unique_ptr<WebAuthRequestSecurityChecker::RemoteValidation>
+ WebAuthRequestSecurityChecker::RemoteValidation::Create(
++    scoped_refptr<network::SharedURLLoaderFactory> shared_url_loader_factory,
+     const url::Origin& caller_origin,
+     const std::string& relying_party_id,
+     base::OnceCallback<void(blink::mojom::AuthenticatorStatus)> callback) {
+@@ -113,6 +114,7 @@ WebAuthRequestSecurityChecker::RemoteValidation::Create(
+ 
+   scoped_refptr<network::SharedURLLoaderFactory> url_loader_factory =
+       GetContentClient()->browser()->GetSystemSharedURLLoaderFactory();
++  url_loader_factory = shared_url_loader_factory;
```

Comment:
I believe this can be replaced with something like
```
#define GetSystemSharedURLLoaderFactory() \
  GetSystemSharedURLLoaderFactory() ? GetUrlLoaderFactory(render_frame_host) : GetUrlLoaderFactory(render_frame_host);
```